rasa_core/channels/slack.py

```diff
@@ -135,6 +137,36 @@ def _is_button_reply(slack_event):
     def _get_button_reply(slack_event):
         return json.loads(slack_event['payload'][0])['actions'][0]['name']
 
+    @staticmethod
+    def _sanitize_user_message(text, uids_to_remove):
+        """
+        Probably a good starting point for pre-formatting of user-provided text,
+        to make NLUs life easier in case they go funky to the power of extreme.
+
+        In the current state will just drop self-mentions of bot itself
+
+        :param text: raw message
+        :param users_to_drop: a list of users to remove from the content
+        :return text:
+        """
+        for uid_to_remove in uids_to_remove:
+            # the seeming duplication turns out to be an enough heuristic
```

Comment:
what do you mean by seeming duplication?

Reply:
the duplication of regexp expressions

Reply:
have tuned that comment to not confuse people